### PR TITLE
Changing repl.it to replit.com

### PR DESCRIPTION
**Pull request by Ashutosh-3601**
Recently repl.it changed to replit.com. Presence for repl.it already existed, just changed it to new domain name.

PS: First PR, if any issues pls let me know :)

### CODE REVIEW

File: websites/R/Replit/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "author": {
+    "name": "Jake.",
+    "id": "480407581085532180"
+  },
+  "service": "Replit.com",
+  "description": {
+    "en": "Replit.com is a San Francisco based start-up and an online IDE (integrated development environment) supporting 40 programming languages, including Python, Lua, JavaScript, Julia, C++, C++11, C, C#, HTML/CSS/JS, Python with turtle and LOLCODE. Repl.it allows users to write code and build apps using a browser, without having to install any software on their devices."
+  },
+  "url": "replit.com",
+  "version": "1.4.5",
```
Comment (EncryptedDev) [inline websites/R/Replit/dist/metadata.json:11]: bump version

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hello, look at my review and provide proof of working

File: websites/R/Replit/presence.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ const presence = new Presence({
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
-      largeImageKey: "repl"
+      largeImageKey: "replit"
```
Comment (StrikerFRFX) [inline websites/R/Replit/presence.ts:7]: If you've changed the `largeImageKey`, you need to change it in the application in the dev portal (which requires you to contact the owner of the presence, which I doubt you have done).

File: websites/R/Replit/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Jake.",
+    "id": "480407581085532180"
+  },
+  "service": "Replit.com",
```
Comment (StrikerFRFX) [inline websites/R/Replit/dist/metadata.json:6]: Service should be `Replit`

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]
Hi @Ashutosh-3601,

Unfortunately your changes have been denied with the reason(s) being:

 - In addition to my requested changes, the Presence simply does not work whilst only changing the domain. The DOMs have changed afaik too, so you would have to go through the process of testing and remaking this presence too.

Also, this one hasn't worked for a while. So, I wish you good luck if you're going to fix. Closing this for now.

Thanks, 
**Striker**, PreMiD Reviewer Team
Contact: me@strikerfrfx.dev, Striker#1337

**Pull request closed without merging**